orchagent/switchorch.cpp

```diff
@@ -1515,3 +1515,42 @@ bool SwitchOrch::querySwitchCapability(sai_object_type_t sai_object, sai_attr_id
         }
     }
 }
+
+// Bind egress ACL table (with bind type switch) to switch
+bool SwitchOrch::bindEgrAclTableToSwitch(sai_object_id_t table_id)
```

Comment:
We could pass the aclStage and make this method common for both ingress and egress ACL.

Reply:
Changed `bindEgrAclTableToSwitch` to `bindAclTableToSwitch` that takes aclStage as input.